src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public static boolean validate(String password, String repeatPassword)
```

Comment:
```suggestion
    public boolean validate(String password, String repeatPassword)
```
we don't need static here